framework/include/systems/NonlinearEigenSystem.h

```diff
@@ -54,6 +54,16 @@ class NonlinearEigenSystem : public NonlinearSystemBase
 
   virtual NumericVector<Number> & RHS() override;
 
+  /*
+   *  A residual vector at MOOSE side for AX
+   */
+  NumericVector<Number> & ResidualVectorAX();
```

Comment:
This method name should start lower case to distinguish it from a class and to be consistent with our style